### PR TITLE
German guide missing image review_10.17

### PR DESCRIPTION
**Pull request by dianaolives**
Missing images in german and english

### CODE REVIEW

File: content/users-guide/cockpit-bundle/data-explorer.md [hunk 1]
```diff
@@ -230,6 +230,7 @@ To use this function, first a dashboard must be created. For details on dashboar
 
 To create a widget from the data explorer of in the navigator, click **More...** in the top menu bar and select **Send as a widget to report** from the context menu.
 
+missing image?
```
Comment (dianaolives) [inline content/users-guide/cockpit-bundle/data-explorer.md:233]: @eniosultan Hi, could you please check if this screenshot is still needed in the English user guide or if the path can be removed? The screenshot itself is missing in the static folder, so I am not sure.

File: content/users-guide/cockpit-bundle/smart-rules.md [hunk 2]
```diff
@@ -145,7 +145,10 @@ To define a threshold rule follow these steps:
 1. In the navigator, select the desired group or device to apply a threshold to.
 2. Switch to the **Data explorer** tab.
 3. If the data point that should raise the threshold is not visible by default, select **Add data point** and [add a data point](#add-data-points).
-4. Click the menu icon at the end of the row of the respective data point and select **Create smart rule**. <br><br> <img src="/images/users-guide/cockpit/cockpit-smart-rules-data-point.png" name="Data point example"/>
+4. Click the menu icon at the end of the row of the respective data point and select **Create smart rule**.
+
+missing image?
```
Comment (dianaolives) [inline content/users-guide/cockpit-bundle/smart-rules.md:150]: @eniosultan Same as mentioned before.

File: content/users-guide/cockpit-bundle/using-widgets.md [hunk 3]
```diff
@@ -100,6 +100,7 @@ On the **Configuration** tab of the widget editor you can use the full text sear
 
 Through the full text search you can find assets in the whole hierarchy, but it requires exact matches, for example, the whole name of the asset.
 
+missing image?
```
Comment (dianaolives) [inline content/users-guide/cockpit-bundle/using-widgets.md:103]: @eniosultan

File: content/users-guide/cockpit-bundle/smart-rules.md (hunk 2 above)
Comment (eniosultan) [reply content/users-guide/cockpit-bundle/smart-rules.md:150]: path can be removed

File: content/users-guide/cockpit-bundle/using-widgets.md (hunk 3 above)
Comment (eniosultan) [reply content/users-guide/cockpit-bundle/using-widgets.md:103]: path can be removed

**Review by MWindel — APPROVED** [review]

**Review by rssag — APPROVED** [review]

**Pull request merged**